source/install/install-rhel-71-postgresql.rst

```diff
@@ -0,0 +1,95 @@
+..  _install-rhel-71-postgresql:
+
+Installing PostgreSQL Database
+==============================
+
+1. Log into the server that will host the database, and open a terminal window.
+
+2. Download the PostgreSQL 9.4 Yum repository.
+
+  ``wget https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-redhat94-9.4-3.noarch.rpm``
+
+3. Install the Yum repository from the file that you downloaded.
+
+  ``sudo yum localinstall pgdg-redhat94-9.4-3.noarch.rpm``
+
+4. Install PostgreSQL.
+
+  ``sudo yum install postgresql94-server postgresql94-contrib``
+
+5. Initialize the database.
+
+  ``sudo service postgresql initdb``
+
+6. Set PostgreSQL to start on boot.
+
+  ``sudo systemctl enable postgresql``
+
+7. Start the PostgreSQL server.
+
+  ``sudo systemctl start postgresql``
+
+8. Switch to the *postgres* Linux user account that was created during the installation.
```

Comment:
Same response as above... I'll update the style guide to italicize user account names.